flood_ml/data_utils.py

```diff
@@ -0,0 +1,64 @@
+"""Data functions for Flood CNN model."""
+
+from constants import GEO_FEATURES, MAP_HEIGHT, MAP_WIDTH, N_FLOOD_MAPS
+
+import numpy as np
+from numpy.lib.stride_tricks import sliding_window_view
+import tensorflow as tf
+
+
+def temporal_window_view(data, window):
+    """Creates a sliding window view over 1D temporal data.
+
+    Args:
+        data: A 2D tensor, where the first axis is the batch dimension and the
+            second axis is the temporal data. Shape: [B, T].
+        window: The window size.
+
+    Returns:
+        A 3D tensor of shape [B, T, window]. In order to maintain the same T,
+        we use zero padding at the beginning of the temporal data.
+    """
+    B = data.shape[0]
+    padding = np.zeros([B, window - 1])
+    data = np.concatenate([padding, data], axis=-1)
+    return sliding_window_view(data, window, axis=-1)
+
+
+def fake_input_batch(
```

Comment:
Is this function useful outside of tests? If not, I'd stick it somewhere like
`flood_ml/testing/data_utils.py`
or
`tests/data_utils.py`
and perhaps giving it a more specific name than 'utils' like `data_gen.py`

Reply:
I wrote this function with testing in mind, but it's currently not being used, so I removed it from this PR.